Baby Monitor/Source Files/Services/MediaPlayer/MicrophoneAmplitudeTracker.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  MicrophoneAmplitudeTracker.swift
+//  Baby Monitor
+
+import Foundation
+import AudioKit
+
+/// A tracker that detects amplitude from the microphone.
+protocol MicrophoneAmplitudeTracker: Any {
+
+    /// The current amplitude with values from 0 to 1.
+    var amplitude: Double { get }
+
+    /// The loudness factor customly counted in order to get poercentage-like result.
+    /// The amlitude multipled by the special factor.
```

Comment:
Some minor typos there. I've recently enabled an IDE plugin to spell-check my code and it helped me to avoid many typos like that. Since then, I think, I am too pedantic about them 🤷‍♂ It saves me a lot of unnecessary time spent on code review through.

```suggestion
    /// The amplitude multiplied by the special factor.
```

Reply:
Nice catch!